src/app/(public)/contact/page.tsx

```
@@ -1,53 +1,48 @@
+          />
+        </div>
+        <div className="space-y-2">
+          <label htmlFor="message" className="block font-medium ">
               Message
-              </label>
-              <TextArea id="message" placeholder="Write your message here..."></TextArea>
-            </div>
-            <Button className="ml-auto w-full text-neutral-1 sm:w-auto">Send message</Button>
-          </form>
+          </label>
+          <TextArea id="message" placeholder="Write your message here..."></TextArea>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)